src/main/resources/latest-spark-swagger-output.yaml

```diff
@@ -964,6 +1005,36 @@ definitions:
       cents:
         type: "integer"
         format: "int32"
+  ItineraryExistence:
+    type: "object"
+    properties:
+      otpRequests:
+        type: "array"
+        items:
+          $ref: "#/definitions/OtpRequest"
+      referenceItinerary:
+        $ref: "#/definitions/Itinerary"
```

Comment:
More non-blocking spark-swagger weirdness: doesn't seem like private fields should be shown.

Reply:
I added code to remove these fields in the public docs in https://github.com/ibi-group/otp-middleware/pull/77/commits/d26baf619770dce9e11667961912b80261de090c.